src/cli/src/cluster/install/k8.rs

```diff
@@ -191,19 +191,27 @@ fn install_core_app(opt: &InstallCommand) -> Result<(), CliError> {
     Ok(())
 }
 
-pub fn install_sys(opt: InstallCommand) {
+pub fn install_sys(opt: InstallCommand) -> Result<(), CliError> {
     helm::repo_add();
     helm::repo_update();
 
-    Command::new("helm")
+    let output = Command::new("helm")
         .arg("install")
         .arg("fluvio-sys")
         .arg("fluvio/fluvio-sys")
         .arg("--set")
         .arg(format!("cloud={}", opt.k8_config.cloud))
-        .inherit();
+        .output()
+        .unwrap();
 
+    if !output.status.success() {
```

Comment:
Can you make changes to Command::inherit() ?  so that this can be applied to all commands?